Cognite.Simulator.Tests/IntegrationTests/ConnectorRuntimeTest.cs

```diff
@@ -93,63 +96,54 @@ private void WriteConfig()
         [Fact]
         public async Task TestConnectorRuntime()
         {
-            var services = new ServiceCollection();
-            services.AddSingleton<DefaultConfig<AutomationConfig>>();
```

Comment:
Since this test still works without the addsingleton line , I assume removing this is fine

Reply:
yes, it was an incorrect place to inject the dependency. it didn't affect the test in any way because this service collection is not used by the runtime of the connector